### PR TITLE
feat/149-frontend

### PR DESCRIPTION
**Pull request by NickSaglioni**
Created Dialog modal component and added it to the delete button of the Manage Access screen.

### CODE REVIEW

File: frontend/src/components/ManageAccess.vue [hunk 1]
```diff
@@ -32,56 +34,56 @@ const userRoleAssignments = ref<UserRoleAssignment[]>([])
...
-            "role_type_code": "C"
-        }
+  {
+    "user_role_xref_id": 1,
+    "user": {
+      "user_type": {
+        code: "I",
+        description: "IDIR",
+      },
+      "user_name": "foo-test"
```
Comment (basilv) [inline frontend/src/components/ManageAccess.vue:44]: Looks like you are reverting a change I need to keep.

File: frontend/src/components/ManageAccess.vue [hunk 2]
```diff
@@ -196,66 +205,69 @@ function uncaughtError() {
 <template>
   <div>
 
-  <PageTitle :displaySelectedApplication=true></PageTitle>
-
```
Comment (basilv) [inline frontend/src/components/ManageAccess.vue:200]: Need to keep PageTitle :displaySelectedApplication=true

**Review by basilv — CHANGES_REQUESTED** [review]
Looks great, just a few issues where you are reverting changes I need to keep.

**Comment by basilv** [comment]
Hey Nick, I'm merging your branch into mine locally so I can control the merge, the PR can be closed (not merged).

File: frontend/src/components/ManageAccess.vue (hunk 1 above)
Comment (NickSaglioni) [reply frontend/src/components/ManageAccess.vue:44]: I am sorry about that, I must have reverted by mistake when fetching the code from server.

**Pull request closed without merging**